src/AutoRest.CSharp/Mgmt/Decorator/SchemaDetection.cs

```diff
@@ -142,16 +142,16 @@ private static RequestParameter GetBodyParameter(ServiceRequest request, Operati
         }
 
         /// <summary>
-        /// Indicates if the given operation group should skip generation or not.
-        /// If the operation group is marked as "SKIP" in configuration, will return false indicating the corresponding container, data... classes should not be generated.
+        /// Returns true if the operation group is a resource; false otherwise (list-only ...).
+        /// If the operation group is marked by <see cref="KeywordToSkipGeneration"/> in configuration, will return false.
         /// </summary>
         /// <param name="og">Operation group.</param>
         /// <param name="config">Management plane configuration.</param>
-        /// <returns></returns>
         public static bool IsResource(this OperationGroup og, MgmtConfiguration config)
```

Comment:
Since we are always testing this is NOT a resource, and then return its opposite, how about we introduce another function `IsNotResource` and this function just returns `!IsNotResource`.
Despite the double negation might introduce a bit confusion to the invocation, we will get more clear logic here.